dev/ci/user-overlays/maximedenes-ssr-merge.sh

```diff
@@ -0,0 +1,4 @@
+if [ $TRAVIS_PULL_REQUEST == "669" ] || [ $TRAVIS_BRANCH == "ssr-merge" ]; then
+    mathcomp_CI_BRANCH=ssr-merge
+    mathcomp_CI_GITURL=https://github.com/maximedenes/math-comp.git
+fi
```

Comment:
I suggest we require the files in this directory to be of the format `XXXX-repos:branch_name.overlay`, example: `0669-maximedenes:ssr-merge.overlay`